src/server.ts

```diff
@@ -96,9 +96,11 @@ export class InversifyExpressServer  {
                     router[metadata.method](metadata.path, ...metadata.middleware, handler);
                 });
 
-                this._app.use(controllerMetadata.path, ...controllerMetadata.middleware, router);
+                this._router.use(controllerMetadata.path, ...controllerMetadata.middleware, router);
             }
         });
+
+        this._app.use("/", this._router);
```

Comment:
Probably hardcoding `"/"` here isn't the best part. Usually APIs have root namespace such as `/api` or smth. So probably additional configuration is needed. What do you guys think about that?